JS/Lesson31/saffet/my-app/src/components/CreateTask/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(task);
+
+    onCreateProps(task);
+    setTask("");
+  }
+
+  return (
+    <>
+      <h2>Create a task in below</h2>
+      <form className="form" action="">
```

Comment:
We usually don't specify "action" attributes when using form elements in React.
```suggestion
      <form className="form" onSubmit={handleSubmit}>
```